### PR TITLE
chore: Quote all value in docker-compose.yml #54

### PR DESCRIPTION
**Pull request by flemzord**

### CODE REVIEW

**Review by paul-nicolas — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -142,8 +142,8 @@ services:
     environment:
       POSTGRES_USER: "formance"
       POSTGRES_PASSWORD: "formance"
-      POSTGRES_MULTIPLE_DATABASES: "auth,ledger,webhooks,payments,wallets"
-      PGDATA: /data/postgres
+      POSTGRES_MULTIPLE_DATABASES: "auth,ledger,webhooks,payments"
```
[the remark this answers is not in the document]
Comment (gfyrag) [reply docker-compose.yml:145]: Wallets does not need a database ;)

File: docker-compose.yml [hunk 2]
```diff
@@ -237,11 +237,11 @@ services:
       search:
         condition: service_healthy
     environment:
-      KAFKA_ADDRESS: redpanda:29092
-      KAFKA_TOPIC: ledger,payments
```
Comment (gfyrag) [inline docker-compose.yml:241]: topics must be separated using a space.

**Review by gfyrag — CHANGES_REQUESTED** [review]
Already there before this PR, but multi values env vars must be separated using a space.

**Review by antoinegelloz — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (gfyrag) [reply docker-compose.yml:241]: also the correct env var is KAFKA_TOPICS

File: docker-compose.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (paul-nicolas) [reply docker-compose.yml:145]: Alright thanks !

**Review by gfyrag — APPROVED** [review]

**Pull request merged**